src/halstead.rs

```diff
@@ -287,16 +330,16 @@ impl Halstead for CppCode {
         let id = node.kind_id();
 
         match id.into() {
-            DOT | LPAREN | COMMA | STAR | GTGT | COLON | Return | Break | Continue | If | Else
-            | Switch | Case | Default | For | While | Goto | Do | Delete | New | Try | Catch
-            | Throw | EQ | AMPAMP | PIPEPIPE | PLUS | PLUSPLUS | SLASH | PERCENT | PIPE | AMP
-            | LTLT | TILDE | LT | LTEQ | EQEQ | BANGEQ | GTEQ | GT | PLUSEQ | BANG | STAREQ
-            | SLASHEQ | PERCENTEQ | GTGTEQ | LTLTEQ | AMPEQ | CARET | CARETEQ | PIPEEQ | LBRACK
-            | LBRACE | QMARK | COLONCOLON | TypeSpecifier | Sizeof => {
+            DOT | LPAREN2 | COMMA | STAR | GTGT | COLON | SEMI | Return | Break | Continue | If
```

Comment:
Why do you remove LPAREN ?
Anyway I just checked the diff between LPAREN and LPAREN2, LPAREN is the parenthesis used in macro definitions (e.g. `#define A(...)` and LPAREN2 in function definition or function call.

Reply:
Oh, my bad. I will re-add that immediately